src/Symfony/Component/Cache/Tests/Adapter/MemcachedAdapterTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Tests\Adapter;
+
+use Symfony\Component\Cache\Adapter\MemcachedAdapter;
+
+/**
+ * @author Rob Frawley 2nd <rmf@src.run>
+ */
+class MemcachedAdapterTest extends AdapterTestCase
+{
+    protected $skippedTests = array(
+        //'testExpiration' => 'Testing expiration slows down the test suite',
+        //'testHasItemReturnsFalseWhenDeferredItemIsExpired' => 'Testing expiration slows down the test suite',
+        //'testDefaultLifeTime' => 'Testing expiration slows down the test suite',
+    );
+
+    private static $client;
+
+    private static function clientDefaultServer()
+    {
+        return array(
+            getenv('MEMCACHED_HOST') ?: '127.0.0.1',
+            getenv('MEMCACHED_PORT') ?: 11211,
+        );
+    }
+
+    public static function setupBeforeClass()
+    {
+        if (!extension_loaded('memcached') || !version_compare(phpversion('memcached'), '2.1.0', '>')) {
+            self::markTestSkipped('Extension memcached >2.1.0 required.');
+        }
+
+        self::$client = new \Memcached();
+        self::$client->addServers(array(static::clientDefaultServer()));
+
+        parent::setupBeforeClass();
+    }
+
+    public function createCachePool($defaultLifetime = 0)
+    {
+        return new MemcachedAdapter(self::$client, str_replace('\\', '.', __CLASS__), $defaultLifetime);
+    }
+
+    /**
+     * @group memcachedAdapter
+     */
+    public function testAdapterAndClientCreationAndServers()
```

Comment:
not sure this test is useful: it tests only test methods/behavior, I'd remove it